### PR TITLE
[doc] correct sql syntax in upgrade.md

### PR DESCRIPTION
**Pull request by tinkerrrr**
## Proposed changes

correct sql syntax in upgrade.md

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [x] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by xy720 — DISMISSED** [review]
LGTM

**Review by chaplinthink — CHANGES_REQUESTED** [review]

File: docs/zh-CN/installing/upgrade.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Doris 可以通过滚动升级的方式，平滑进行升级。建议按照以
 	$ mysql-client > admin set frontend config("disable_balance" = "true");
 
 	# 关闭 colocation 表的副本均衡逻辑。关闭后，不会再出发 colocation 表的副本重分布操作。
```
[the remark this answers is not in the document]
Comment (tinkerrrr) [reply docs/zh-CN/installing/upgrade.md]: Done

**Review by chaplinthink — APPROVED** [review]

**Review by morningman — COMMENTED** [review]
LGTM

**Review by hf200012 — APPROVED** [review]
LGTM

**Pull request merged**